crates/holochain/src/conductor/conductor.rs

```diff
@@ -267,7 +268,7 @@ pub struct Conductor {
 
     /// File system and in-memory cache for wasmer modules.
     // Used in ribosomes but kept here as a single instance.
```

Comment:
Maybe update doc comment explaining why this is optional?

Reply:
Though I guess it'd be valid to run without it with `wasmer_sys` too

Reply:
expanded the comment and made it clearer